src/lib/database/database-manager.ts

```diff
@@ -0,0 +1,100 @@
+import Database from 'better-sqlite3';
+
+export default class DatabaseManager {
```

Comment:
Not really required, but leaving as an example of abstraction during dev. I'll excise it, or reduce its scope later.